pyrft/random_field_generation.py

```diff
@@ -44,8 +49,8 @@ def smooth(data, fwhm, mask = 0):
     # Calculate the standard deviation from the fwhm
     sigma = pr.fwhm2sigma(fwhm)
 
-    for I in np.arange(data.fibersize):
-        data.field[...,I] = gaussian_filter(data.field[...,I]*data.mask, sigma = sigma)*data.mask
+    for i in np.arange(data.fibersize):
+        data.field[...,i] = gaussian_filter(data.field[...,i]*data.mask, sigma = sigma)*data.mask

```

Comment:
```suggestion
        data.field[...,i] = gaussian_filter(data.field[...,i] * data.mask, sigma=sigma) * data.mask

```

Reply:
Did you mean to put data.mask on a separate line for this?

Reply:
I simply mean to add white space whenever needed. If the line becomes too long, you can break it
```suggestion
        data.field[...,i] = gaussian_filter(data.field[...,i] * data.mask, sigma=sigma)\
                                 * data.mask

```

Reply:
Okay thanks!